report-pox-krypton.js

```diff
@@ -310,8 +310,31 @@ const staging_db_path = 'chainstate/chain-00000080-testnet/blocks/staging.db'
 //                                            vtxindex INT NOT NULL
 //     );
 
-const data_root_path = `${root}${process.argv[3] || process.argv[2]}`
-const use_txs = process.argv[2] === '-t'
+// const data_root_path = `${root}${process.argv[3] || process.argv[2]}`
+// const use_txs = process.argv[2] === '-t'
+
+let use_txs = false;
+let use_csv = false;
+var myArgs = process.argv.slice(2);
+// console.log('myArgs: ', myArgs);
+
+for (let j = 0; j < myArgs.length; j++){
+  switch (myArgs[j]) {
+    case '-t':
+      use_txs = true;
+      break;
+    case '-csv':
+      use_csv = true;
+      break;
+    default:
+      if (myArgs[j].startsWith('/tmp')) {
+        var data_root_path = `${root}${myArgs[j]}`;
```

Comment:
definitely no `var` used sneakily like this...  Define `data_root_path` with `let` above loop, and initialize here.

Reply:
Sneakiness removed in f7c73d2,